model/AI/Classes/AIPackage.md

```diff
@@ -0,0 +1,107 @@
+-------
+SPDX-License-Identifier: Community-Spec-1.0
+
+# AIPackage
```

Comment:
this file is missing the summary section. this might break parsing

Reply:
Thanks. Added summary to the new version